jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/PerspectiveIds.java

```diff
@@ -23,4 +23,18 @@
     String SEARCH_PARAMETER_TASK_ID = "taskId";
     String SEARCH_PARAMETER_JOB_ID = "jobId";
     String SEARCH_PARAMETER_IS_ERROR_ACK = "isErrorAck";
-}
+
+    String PROCESS_INSTANCE_DETAILS_SCREEN = "Process Instance Details Multi";
+    String PROCESS_INSTANCE_LIST_SCREEN = "DataSet Process Instance List With Variables";
+    String PROCESS_DEFINITION_DETAILS_SCREEN = "Advanced Process Details Multi";
+    String PROCESS_DEFINITION_LIST_SCREEN = "Process Definition List";
+    String DASHBOARD_SCREEN = "DashboardScreen";
+    String EXECUTION_ERROR_DETAILS_SCREEN = "Execution Error Details";
+    String EXECUTION_ERROR_LIST_SCREEN = "Execution Error List";
+    String JOB_LIST_SCREEN = "Job List";
+    String JOB_DETAILS_SCREEN = "Job Details";
+    String TASK_ADMIN_LIST_SCREEN = "Task Admin List";
+    String TASK_LIST_SCREEN = "Task List";
+    String TASK_DETAILS_SCREEN = "Task Details Multi";
+
```

Comment:
@cristianonicolai Looks good. Have just one question regarding keeping the names for some identifiers, which look a bit cryptic (e.g. ```"DataSet Process Instance List With Variables"```) . I see also a couple of others which might use some trimming and aligning with the rest.
Is this the place and time to address something like this or are these about to change anyway moving to new perspectives in the future?

Reply:
@bsremac @cristianonicolai maybe some literal could be more explanatory and maybe is a good moment to unify with the other workbench screens.

Reply:
I agree, now that we combine these ids, it makes easier to visualize the inconsistency. I would be fine for us to update it now. In general, I see the following patterns: append Screen as suffix and remove any space. Will do a quick update and you let me know what you think.